src/envoy/mixer/http_filter.cc

```diff
@@ -566,18 +545,39 @@ class MixerConfigFactory : public NamedHttpFilterConfigFactory {
                                           FactoryContext& context) override {
     Http::Mixer::ConfigPtr mixer_config(
         new Http::Mixer::Config(config, context));
-    return
-        [mixer_config](Http::FilterChainFactoryCallbacks& callbacks) -> void {
-          if (mixer_config->auth_config()) {
-            callbacks.addStreamDecoderFilter(Http::StreamDecoderFilterSharedPtr{
-                new Http::JwtVerificationFilter(mixer_config->auth_config())});
-          }
-          std::shared_ptr<Http::Mixer::Instance> instance =
-              std::make_shared<Http::Mixer::Instance>(mixer_config);
-          callbacks.addStreamDecoderFilter(
-              Http::StreamDecoderFilterSharedPtr(instance));
-          callbacks.addAccessLogHandler(AccessLog::InstanceSharedPtr(instance));
-        };
+
+    HttpFilterFactoryCb auth_filter_cb;
+    auto auth_factory =
+        Registry::FactoryRegistry<NamedHttpFilterConfigFactory>::getFactory(
+            "jwt-auth");
+    if (auth_factory) {
+      auto auth_config = mixer_config->auth_config();
+      if (auth_config) {
+        auto proto_config = auth_factory->createEmptyConfigProto();
```

Comment:
Use `Config::Utility::translateToFactoryConfig`

Reply:
Could not use that function. It assume the input proto has a "config" field.   My input proto is EndUserAuthenticationPolicySpec.  it doesn't have "config" on top of it.